schemas/api/appIntent.schema.json

```
@@ -1,17 +1,17 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "$id": "https://fdc3.finos.org/schemas/next/appIntent.schema.json",
+    "$id": "https://fdc3.finos.org/schemas/next/api/appIntent.schema.json",
     "title": "AppIntent",
     "description": "Represents the binding of an intent to apps",
     "type": "object",
     "properties": {
         "intent": {
-            "$ref": "https://fdc3.finos.org/schemas/next/intentMetadata.schema.json"
+            "$ref": "intentMetadata.schema.json#"
```

```suggestion
            "$ref": "intentMetadata.schema.json"
```
inconsistent? not sure if it does anything anyway